test/NuGet.Core.Tests/NuGet.Common.Test/LocalizedTestCollection.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    [CollectionDefinition(TestName, DisableParallelization = true)]
+    internal static class LocalizedTestCollection
+    {
+        public const string TestName = "LocalizedTests";
+
+        private static CultureInfo DefaultCurrentUICulture;
+
+        private static CultureInfo DefaultCurrentCulture;
+
+        private static readonly object LockObject = new();
+
+        static LocalizedTestCollection()
+        {
+            lock (LockObject)
```

Comment:
I don't think we need this, as you could just initialize the properties directly?

> No additional locking mechanisms are needed in the body of a static constructor.

https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/static-constructors#remarks

Reply:
Good point. Didn't notice it. I just wanted to implement mutual exclusion when getting / setting these properties.

Reply:
I am going to remove it. It's very difficult to guarantee a locale since threads can be recycled between test executions. See https://github.com/xunit/xunit/issues/1605#issuecomment-488382244